src/charm.py

```diff
@@ -209,7 +210,18 @@ def _on_attach_ue_action(self, event: ActionEvent) -> None:
         service_restart(SRS_UE_SERVICE)
         self._stored.ue_attached = True
         self.unit.status = self._get_current_status()
-        event.set_results({"status": "ok", "message": "Attached successfully"})
+        if ue_ip := get_iface_ip_address("tun_srsue"):
+            event.set_results(
+                {"status": "ok", "message": "Attached successfully", "ue-ipv4": ue_ip}
+            )
+        else:
+            event.set_results(
+                {
+                    "status": "ok",
+                    "message": "Attached successfully",
```

Comment:
is successful in this case?

Reply:
I wanted to discuss this, shouldn't we merge the 3 fields into one? Something like:
When successful
```python
{
	"status": "ok",
"message": "Attached successfully",
                    "ue-ipv4": "No UE ip found, please make sure the interface is up.",
                }
```

Reply:
I think I didn't get it at first, I change it a bit. Let me know what you think.